.gitlab/package_build/linux.yml

```diff
@@ -90,33 +90,20 @@ datadog-agent-7-arm64-fips:
     - ls -la $OMNIBUS_PACKAGE_DIR
     - !reference [.upload_sbom_artifacts]
 
-iot-agent-x64:
-  extends: .iot-agent-common
+.iot-otel-statsd-x64:
   tags: ["arch:amd64", "specific:true"]
   image: registry.ddbuild.io/ci/datadog-agent-buildimages/linux$CI_IMAGE_LINUX_SUFFIX:$CI_IMAGE_LINUX
   variables:
     DD_CC: "x86_64-unknown-linux-gnu-gcc"
     DD_CXX: "x86_64-unknown-linux-gnu-g++"
 
-iot-agent-arm64:
-  extends: .iot-agent-common
+.iot-otel-statsd-arm64:
   tags: ["arch:arm64", "specific:true"]
   image: registry.ddbuild.io/ci/datadog-agent-buildimages/linux$CI_IMAGE_LINUX_SUFFIX:$CI_IMAGE_LINUX
   variables:
     DD_CC: "aarch64-unknown-linux-gnu-gcc"
     DD_CXX: "aarch64-unknown-linux-gnu-g++"
```

Comment:
Naming nit: what about `.toolchain-common-x64` or `build-x64` or something like that ?
I was initially very confused when I saw `iot-otel-statsd` 😄